modules/ding_dibs/ding_dibs.email_reports.inc

```diff
@@ -99,7 +99,7 @@ function ding_dibs_email_reports() {
     // "dibs_capture_status" have both equally have the status of unpaid (0)
     // or paid (1).
     // Testing conducted by feedback from.
-    if (!(
+    if ((
```

Comment:
Suggest to refactor this check.
It has a weirdo indent which has an impact on readability.